configure.ac

```diff
@@ -662,7 +662,7 @@ fi
 # Check for stream support via WinPR
 if test "x${have_freerdp}" = "xyes"
 then
-    AC_CHECK_HEADER(winpr/stream.h,,
+    AC_CHECK_HEADER(winpr/stream.h,[RDP_LIBS="$RDP_LIBS -lwinpr-utils"],
```

Comment:
So, the rationale for adding it with the streams.h section is that this library (winpr-utils) is where the Stream_New and Stream_Free functions actually exist.  If you look at the original ticket, you'll see that when you do not link against winpr-utils, you get two undefined symbols (Stream_New and Stream_Free, in addition to the odd third undefined one) - when you link against winpr-utils, these two symbols are defined.

I'll try to take a look at reworking it in with the final determination of WinPR presence later on, and split out WINPR_LIBS from RDP_LIBS and find the right places in the src/protocols/rdp/Makefile.am to put that in.